### PR TITLE
BC-570 add cachet

### PR DESCRIPTION
**Pull request by alriese**
# Description

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: ansible/roles/cachet/templates/svc.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  type: ClusterIP
+  ports:
+    - name: cachet8000
+      port: 8000
+      targetPort: 8000
+      protocol: TCP
+    - name: cachet80
+      port: 80
+      targetPort: 80
+      protocol: TCP
```
Comment (mamutmk5) [inline ansible/roles/cachet/templates/svc.yml.j2:18]: The cachet pod don't have an port 80 form external.
At our  casse it woud be handeld by an ingress. ( but not in this step )

File: ansible/roles/cachet/templates/svc.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+kind: Service
+metadata:
+  name: cachet-svc
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: cachet
+spec:
+  type: ClusterIP
+  ports:
+    - name: cachet8000
```
Comment (mamutmk5) [inline ansible/roles/cachet/templates/svc.yml.j2:11]: name shoud be like  the name of the container port at the deployment in this casse it where _cachet_

File: ansible/roles/cachet/templates/configmap.yml.j2 [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+kind: ConfigMap
+metadata:
+  name: cachet-configmap
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: cachet
+data:
+  # general
+  DB_DRIVER: "pgsql"
+  SESSION_DRIVER: "file"
```
Comment (mamutmk5) [inline ansible/roles/cachet/templates/configmap.yml.j2:11]: what else session drivers are posible?

**Review by mamutmk5 — CHANGES_REQUESTED** [review]
Please set for each hostgroup ( default, nbc, brb ) the env vars:
FEATURE_ALERTS_ENABLED: true
ALERT_STATUS_URL: http://cachet-svc:8000

File: ansible/roles/cachet/templates/configmap.yml.j2 (hunk 3 above)
Comment (alriese) [reply ansible/roles/cachet/templates/configmap.yml.j2:11]: The current values are just for testing purpose. The following drivers are possible: 
**session and cache drivers**: APC(U), Array, Database, File, Memcached, Redis.
**queue drivers**: None, Synchronous, Database, Beanstalk, Amazon SQS, Redis
**mail drivers**: SMTP, Mail, Sendmail, Mailgun, Mandrill, Amazon SES, SparkPost, Log

**Pull request closed without merging**